x-pack/plugins/infra/public/components/asset_details/tabs/common/popover.tsx

```diff
@@ -27,7 +27,13 @@ export const Popover = ({
     <EuiPopover
       panelPaddingSize="s"
       button={
-        <button onClick={togglePopover} data-test-subj={props['data-test-subj']}>
+        <button
+          onClick={(e) => {
+            e.stopPropagation();
+            togglePopover();
+          }}
```

Comment:
Following the comment above, you could go back to the way it was:

```suggestion
          onClick={togglePopover}
```

Reply:
ah my bad, I just thought it was using the `<LinkToAlertsRule` component